deploy/manifests/dev/us-east-2/tenant/storetheindex/indexstar/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             - '--backends=http://cali-indexer:3000/'
             - '--backends=http://ago-indexer:3000/'
             - '--backends=http://dhstore.internal.dev.cid.contact/'
-            - '--backends=http://dhfind.internal.dev.cid.contact/'
+            - '--backends=http://dhfind:40080/'
```

Comment:
`dhfind` service is [patched](https://github.com/ipni/storetheindex/blob/main/deploy/manifests/dev/us-east-2/tenant/storetheindex/dhfind/service.yaml#L15) in dev overlay to be an AWS NLB.

If you'd like to experiment with this, I recommend creating a new `Service` object in dev overlay like [this](https://github.com/ipni/storetheindex/blob/main/deploy/manifests/base/storetheindex-single/service.yaml#L15) with type `ClusterIP` and use that.